src/Uno.Extensions.Navigation.UI/Navigator.cs

```diff
@@ -531,6 +531,18 @@ protected virtual async Task<bool> RegionCanNavigate(Route route, RouteInfo? rou
 			}
 		}
 
+		if (!(Route?.IsEmpty()??true))
+		{
+			var currentRouteMap = Resolver.FindByPath(Route?.Base);
```

Comment:
If `Route` was null, we wouldn't be in this branch

```suggestion
			var currentRouteMap = Resolver.FindByPath(Route.Base);
```